Add deprecation warning for App Container (#8475)
zeit/next.js: errors/app-container-deprecated.md

# App Container Deprecated

#### Why This Error Occurred

> In versions prior to v9.0.4-canary.3 the `Container` component was used in `_app` to handle scrolling to hashes. Now this handling has been moved up the tree so `Container` is no longer needed in `_app`.

Reviewer: ```suggestion
In versions prior to v9.0.4 the `<Container>` component was used in `./pages/_app.js` to handle scrolling to hashes.
This handling has been moved up the tree so the `<Container>` component is no longer needed in your custom `<App>`!
```

> 

Reviewer: ```suggestion
```

> You should remove the `Container` component from your `_app` since it a no-op now and will be removed in future versions.

Reviewer: ```suggestion
```

#### Possible Ways to Fix It

> Remove the `Container` component from `_app`

Reviewer: ```suggestion
Remove the `<Container>` component from your Custom `<App>` (`./pages/_app.js`).
```
